website-next/components/projects/projects-page-component.tsx

```diff
@@ -35,6 +35,45 @@ const FlexNavItem = styled('div')<navItemProps>(
     color: props.selected ? 'white' : colors.default,
   }),
 )
+
+interface sortButtonProps {
```

Comment:
```suggestion
interface SortButtonProps {
```

Please always use a capital letter at the start of any types and interfaces (aka [PascalCase](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841))

You'll also need to update the use below